os/kernel/binary_manager/binary_manager.h

```diff
@@ -149,7 +149,7 @@ struct binmgr_uinfo_s {
 	load_attr_t load_attr;
 	uint8_t load_priority[FILES_PER_BIN];
 	uint32_t bin_ver[FILES_PER_BIN];
-	char kernel_ver[KERNEL_VER_MAX];
+	float kernel_ver;
```

Comment:
The names are a little confusing 
Currently we are using >
**kernel_ver** for "User binary data type in binary table"
**version** for "Kernel binary data type in kernel table"
I feel we can have the naming something like kversion_ktable & kversion_btable
What is your opinion ?

Reply:
kernel and user table is already separated and this is structure for user binary.
User binary table also should keep kernel version for checking compatibility of kernel version.